docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,451 @@
+ 
+Refer to [API Reference](https://aws-controllers-k8s.github.io/community/reference/) for *EC2*
+to find the supported reference fields.
+{{% /hint %}}
+ 
+ 
+The output should look similar to:
+```
+vpc.ec2.services.k8s.aws/ack-vpc created
+internetgateway.ec2.services.k8s.aws/ack-igw created
+routetable.ec2.services.k8s.aws/ack-public-route-table created
+routetable.ec2.services.k8s.aws/ack-private-route-table created
+natgateway.ec2.services.k8s.aws/ack-natgateway created
+elasticipaddress.ec2.services.k8s.aws/ack-eip created
+subnet.ec2.services.k8s.aws/ack-public-subnet created
+subnet.ec2.services.k8s.aws/ack-private-subnet created
+securitygroup.ec2.services.k8s.aws/ack-security-group created
+```
+ 
+* Check the **Custom Resource's** using `kubectl describe`:
+```
+kubectl describe vpcs
+kubectl describe internetgateways
+kubectl describe routetables
+kubectl describe natgateways
+kubectl describe elasticipaddresses
+kubectl describe subnets
+kubectl describe securitygroups
+```
+ 
+* Subnet gets into an 'available' state with a `ACK.ReferencesResolved = True` condition attached notifying users that the references (VPC, RouteTable) have been found and resolved:
+ 
+```
+Status:
+  Ack Resource Metadata:
+    Arn:                       arn:aws:ec2:us-west-2:<ID>:subnet/subnet-0ba22f5820bb41584
+    Owner Account ID:          <ID>
+    Region:                    us-west-2
+  Available IP Address Count:  4091
+  Conditions:
+    Last Transition Time:           2022-10-13T14:54:39Z
+    Status:                         True
+    Type:                           ACK.ReferencesResolved
+    Last Transition Time:           2022-10-13T14:54:41Z
+    Message:                        Resource synced successfully
+    Reason:
+    Status:                         True
+    Type:                           ACK.ResourceSynced
+  Default For AZ:                   false
+  Map Customer Owned IP On Launch:  false
+  Owner ID:                         515336597380
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-<ID>
+ 
+```
+ 
+### Validate
+ 
+This network setup should allow Instances deployed in the Private Subnet to connect to the internet. To validate this behavior deploy an Instance into the Private Subnet and the Public Subnet (bastion host). After deployments, `ssh` into the bastion host, then `ssh` into the Private Subnet Instance, and test internet connection. Security group is required by both instances launched in public and private subnets.
+ 
+* Deploy an Instance into the Private Subnet:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: ack-instance-private
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2; us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<private-ID>
+  securityGroupIDs:
```

Comment:
Can we use more references here?

Reply:
From the instance CRD, I can see that there is no work around for references as of now, we need to manually update the ID's in the manifest.

Reply:
can you explicitly call out that this field needs to be updated/copied over?